### DISCUSSION TITLE
BroadCast

### DISCUSSION THREAD

**Discussion by xRedKurama** [Zrips/CMI#9049] [Q&A]
Hello! I had this situation: I wanted to change the label in the broadcast chat files, but I can't find the path to the file with the name of the label in the chat. I hope for your help!

**Comment by CloudeLecaw** [comment]
You finde it in cmis translation file at the broadcast command..

![07-10-2024_22-50-29](https://github.com/user-attachments/assets/2b183724-9c6a-4e70-bd9a-323d477c0b52)


Option two would be to design a own lable for the single broadcast staring the broadcastmessage with !
example;
`cmi broadcast !&b&l█&c  Protipp  &b&l█  &eHello!`
